subsys/lorawan/lorawan.c

```diff
@@ -1,18 +1,20 @@
 /*
  * Copyright (c) 2020 Manivannan Sadhasivam <mani@kernel.org>
+ * Copyright (c) 2022 Intellinium <giuliano.franchetto@intellinium.com>
```

Comment:
IANAL but I tend to put copyright only if it is a new file or I've made some significant changes to the existing file, like heavy refactoring etc,..

Reply:
what is the rule to choose between ```#if defined()``` and ```IS_ENABLED``` ?

Reply:
Don't know if there is an explicit rule. But I would use `IS_ENABLED` for small sections within a function (as it makes reading the code more smooth) and `#if defined` for larger sections of code (e.g. entire functions) that need to be selected depending on the config.